tests/config.go

```diff
@@ -55,6 +59,11 @@ type Nodes struct {
 // NewConfig creates a new config.
 func NewConfig() (*Config, error) {
 	cfg := &Config{
+
+		PDMaxReplicas:       5,
```

Comment:
You may set `PDMaxReplicas ` default to `3`. This is the number of replicas for each region.

Reply:
This config is what we update the ConfigMap **to**, maybe the variable name should more concise.